superset/models/core.py

```diff
@@ -1071,9 +1069,22 @@ def get_oauth2_config(self) -> OAuth2ClientConfig | None:
         return self.db_engine_spec.get_oauth2_config()
 
 
-sqla.event.listen(Database, "after_insert", security_manager.database_after_insert)
```

Comment:
Alternatively we can move these registrations into here: https://github.com/apache/superset/blob/master/superset/initialization/__init__.py#L414

I'm personally not in favour of stuff happening at the root level of modules

Reply:
I like the idea of centralizing them there, but screams circular imports, let me see